scripts/update-integration-test-coordinates.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+python integration-test.py -printcoords | python scripts/update-integration-test-coordinates.py
```

Comment:
Is `integration-test.py` in the current working directory? Should we use the script location to make sure that it finds the right file?